maps/metaclub.dmm

```diff
@@ -6266,13 +6266,13 @@
 "cqz" = (/obj/structure/disposalpipe/segment,/turf/simulated/floor{icon_state = "white"},/area/science/lab)
 "cqA" = (/obj/structure/bed/chair/office/light{dir = 4},/turf/simulated/floor{icon_state = "white"},/area/science/lab)
 "cqB" = (/obj/item/weapon/reagent_containers/glass/beaker/large{pixel_x = -3; pixel_y = 3},/obj/item/weapon/reagent_containers/glass/beaker{pixel_x = 8; pixel_y = 2},/obj/item/weapon/reagent_containers/glass/beaker{pixel_x = 8; pixel_y = 2},/obj/item/weapon/reagent_containers/dropper,/obj/structure/table,/obj/machinery/light_switch{pixel_x = 27},/obj/machinery/camera{c_tag = "Science - Research and Development"; dir = 8; network = list("SS13","RD")},/turf/simulated/floor{dir = 4; icon_state = "whitepurple"},/area/science/lab)
-"cqC" = (/obj/machinery/atmospherics/pipe/simple/supply/hidden,/obj/structure/disposalpipe/segment,/obj/machinery/door/firedoor/border_only,/obj/machinery/door/airlock/glass_medical{glass = 0; icon = 'icons/obj/doors/Doorresearch.dmi'; id_tag = ""; name = "Research Division"; opacity = 1; req_access_txt = "0"; req_one_access_txt = "47"},/obj/effect/decal/warning_stripes{icon_state = "unloading"},/turf/simulated/floor{dir = 4; icon_state = "floor"},/area/science/hallway{name = "Research Division"})
-"cqD" = (/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden,/obj/machinery/door/firedoor/border_only,/obj/machinery/door/airlock/glass_medical{glass = 0; icon = 'icons/obj/doors/Doorresearch.dmi'; id_tag = ""; name = "Research Division"; opacity = 1; req_access_txt = "0"; req_one_access_txt = "47"},/obj/effect/decal/warning_stripes{icon_state = "unloading"},/turf/simulated/floor{dir = 4; icon_state = "floor"},/area/science/hallway{name = "Research Division"})
```

Comment:
I have no idea why these changes were made but PJB says it's alright.  Please review this and make sure smartgit didn't fuck my shit up.